### PR TITLE
Updates dev dependencies

### PR DESCRIPTION
**Pull request by andrewtpoe**
## OVERVIEW
Updates the dev dependencies to the latest available versions.

## WHERE SHOULD THE REVIEWER START?
`package.json`

## HOW CAN THIS BE MANUALLY TESTED?
Run `yarn review` and verify lint and tests pass

## ANY NEW DEPENDENCIES ADDED?
No additions, but several updates.

## CHECKLIST
_Be sure all items are_ ✅ _before submitting a PR for review._
* [x] Verify the linter and tests pass: `npm run review`
* [x] Verify this branch is rebased with the latest master

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,7 +17,9 @@
     "css",
     "react"
   ],
-  "contributors" : [ "UI Platform Development Team at Decisiv, Inc." ],
+  "contributors": [
+    "UI Platform Development Team at Decisiv, Inc."
```
Comment (alanbsmith) [inline package.json]: Team Dewey.

**Comment by andrewtpoe** [comment]
It should work fine with v1. I don't know that it is specifically necessary to list as a peer dep though. I think of peer deps as "this won't function with that". Technically, nothing in this lib is directly connected to styled-components functions. It's just designed to work with how SC calls things.

File: package.json [hunk 2]
```diff
@@ -47,14 +47,14 @@
   },
   "devDependencies": {
     "babel-cli": "^6.24.1",
```
Comment (alanbsmith) [inline package.json]: looks like `babel-cli` can also be updated to 6.26.0

File: package.json [hunk 3]
```diff
@@ -47,14 +47,14 @@
...
-    "eslint": "^3.19.0",
-    "eslint-config-airbnb-base": "^11.2.0",
+    "eslint": "^4.8.0",
+    "eslint-config-airbnb-base": "^12.0.2",
     "eslint-plugin-import": "^2.7.0",
-    "jest": "^20.0.4",
+    "jest": "^21.2.1",
     "jest-junit": "^3.1.0",
     "opn-cli": "^3.1.0",
     "prop-types": "^15.5.10",
```
Comment (alanbsmith) [inline package.json]: I think `prop-types` can go up to 15.6.0

Comment (alanbsmith) [reply package.json]: it also looks like `rimraf` could go up to 2.6.2

Comment (andrewtpoe) [reply package.json]: Strange, but `yarn outdated` didn't report those for me. Oh well, updated them now.

**Pull request merged**